.gitignore

```
@@ -1,4 +1,5 @@
 /coverage.txt
+.output
```

It matches the sub-directories as well.
```suggestion
/.output/
```